packages/api-gateway-service/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "api-gateway",
+  "description": "API Gateway for One Platform",
+  "opservice": {
+    "className": "ApiGateway",
+    "serviceName": "api-gateway"
+  },
+  "version": "0.0.1",
+  "keywords": [
+    "web-components",
+    "html"
```

Comment:
Are you using web-components and html in this package?

Reply:
updated the tags